perception/traffic_light_classifier/src/cnn_classifier.cpp

```diff
@@ -39,10 +39,14 @@ CNNClassifier::CNNClassifier(rclcpp::Node * node_ptr) : node_ptr_(node_ptr)
   input_c_ = node_ptr_->declare_parameter("input_c", 3);
   input_h_ = node_ptr_->declare_parameter("input_h", 224);
   input_w_ = node_ptr_->declare_parameter("input_w", 224);
+  mean_ = node_ptr_->declare_parameter("mean", std::vector<double>({0.242, 0.193, 0.201}));
+  std_ = node_ptr_->declare_parameter("std", std::vector<double>({1.0, 1.0, 1.0}));
+  input_name_ = node_ptr_->declare_parameter("input_name", std::string("input_0"));
+  output_name_ = node_ptr_->declare_parameter("output_name", std::string("output_0"));
```

Comment:
I think it doesn't need to be a private variable.
```suggestion
  input_name = node_ptr_->declare_parameter("input_name", std::string("input_0"));
  output_name = node_ptr_->declare_parameter("output_name", std::string("output_0"));
```